internal/config/seccomp/notifier.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/cri-o/cri-o/internal/log"
 	"github.com/cri-o/cri-o/pkg/annotations"
```

Comment:
IMHO it'd be better to remove v1 completely.

Reply:
Done